tests/testthat/test-message.R

```diff
@@ -8,6 +8,13 @@ test_that("msg works", {
         "there are 3 documents"
     )
     
+    # NULL indices
+    expect_equal(
+        quanteda:::msg("there are %d %s", list(3, "documents"),
+                       indecies = list(NULL, NULL)),
```

Comment:
third argument should be `indices`. Let's add the `check_dots()` to all functions that take `...`. `msg()` does not call that...

Reply:
Let's not use `check_dots()` in internal functions.